<?xml version="1.0"?>
<launch>

    <!-- Load parameters -->
    <rosparam command="load" ns="" file="$(find table_plane_extractor)/config/config.yaml"/>
    <node name="get_objects_on_table" pkg="table_plane_extractor" type="get_objects_on_table.py"  output="screen"/>
</launch>